src/System.Diagnostics.TextWriterTraceListener/tests/CtorsStreamTests.cs

```diff
@@ -1,143 +1,115 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
+using System.Collections.Generic;
 using System.IO;
-using System.Diagnostics;
 using Xunit;
 
 namespace System.Diagnostics.TextWriterTraceListenerTests
 {
     public class CtorsStreamTests
     {
-        private TextWriterTraceListener _textWriter;
-        private Stream _stream;
-        private string _fileName;
-
         [Fact]
-        public void Test01()
+        public void NegativeTests()
         {
-            //[] Stream is null
-            _stream = null;
-            Assert.Throws<ArgumentNullException>(delegate () { new TextWriterTraceListener(_stream); });
+            Assert.Throws<ArgumentNullException>(() => new TextWriterTraceListener(stream: null));
 
-            //[] Stream is closed
-            _stream = new MemoryStream();
-            _stream.Dispose();
-            Assert.Throws<ArgumentException>(delegate () { new TextWriterTraceListener(_stream); });
+            var closedStream = new MemoryStream();
+            // We should call Close method here but for some reason we receive
+            // error CS1061: 'System.IO.MemoryStream' does not contain a definition for 'Close'.
+            // Calling Dispose should have same effect as calling Close.
```

Comment:
@karelz, amended so comments got deleted.
FYI you can leave persistent comments (stable for amending/rebasing)  through the "File Changes" tab on this page.